OamLog+forwardings.yaml

```diff
@@ -0,0 +1,520 @@
+          uuid: ol-1-0-0-op-s-bm-002
+      fc-port-update:
+        - server-name: /v1/redirect-service-request-information
+          uuid: ol-1-0-0-op-s-bm-002
+      fc-port-deletion:
+      operation-client-deletion:
+    initiating-requests:
+      - server-name: /v1/register-yourself
+        uuid: ol-1-0-0-op-s-bm-000
+      - server-name: /v1/embed-yourself
+        uuid: ol-1-0-0-op-s-bm-001
+      - server-name: /v1/redirect-service-request-information
+        uuid: ol-1-0-0-op-s-bm-002
+      - server-name: /v1/redirect-oam-request-information
+        uuid: ol-1-0-0-op-s-bm-003
+      - server-name: /v1/end-subscription
+        uuid: ol-1-0-0-op-s-bm-004
+      - server-name: /v1/inquire-oam-request-approvals
+        uuid: ol-1-0-0-op-s-bm-005
+      - server-name: /v1/update-client
+        uuid: ol-1-0-0-op-s-bm-007
+      - server-name: /v1/list-ltps-and-fcs
+        uuid: ol-1-0-0-op-s-bm-008
+      - server-name: /v1/redirect-topology-change-information
+        uuid: ol-1-0-0-op-s-bm-009
+      - server-name: /v1/update-operation-key
+        uuid: ol-1-0-0-op-s-bm-010
+      - server-name: /v1/update-operation-client
+        uuid: ol-1-0-0-op-s-bm-011
+      - server-name: /v1/bequeath-your-data-and-die
+        uuid: ol-1-0-0-op-s-im-000
+      - server-name: /v1/start-application-in-generic-representation
+        uuid: ol-1-0-0-op-s-bs-000
+      - server-name: /v1/inform-about-application
+        uuid: ol-1-0-0-op-s-bs-001
+      - server-name: /v1/inform-about-application-in-generic-representation
+        uuid: ol-1-0-0-op-s-bs-002
+      - server-name: /v1/inform-about-release-history
+        uuid: ol-1-0-0-op-s-bs-003
+      - server-name: /v1/inform-about-release-history-in-generic-representation
+        uuid: ol-1-0-0-op-s-bs-004
+      - service-name: /v1/regard-application
+        uuid: ol-1-0-0-op-s-is-001
+      - service-name: /v1/disregard-application
+        uuid: ol-1-0-0-op-s-is-002
+      - service-name: /v1/list-applications
+        uuid: ol-1-0-0-op-s-is-003
+      - service-name: /v1/record-oam-request
+        uuid: ol-1-0-0-op-s-is-004
+      - service-name: /v1/list-records
+        uuid: ol-1-0-0-op-s-is-005
+      - service-name: /v1/list-records-of-application
+        uuid: ol-1-0-0-op-s-is-006
+    consequent-requests:
+      - client-name: ExecutionAndTraceLog://v1/record-service-request
+        uuid: ol-1-0-0-op-c-bs-ol-1-0-0-000
+
+  - forwarding-name: OamRequestCausesLoggingRequest
+    uuid: ol-1-0-0-op-fc-bm-003
+    forwarding-type: InvariantProcessSnippet
+    management-requests:
+      operation-client-update:
+        - server-name: /v1/redirect-oam-request-information
+          uuid: ol-1-0-0-op-s-bm-003
+      fc-port-update:
+        - server-name: /v1/redirect-oam-request-information
+          uuid: ol-1-0-0-op-s-bm-003
+      fc-port-deletion:
+      operation-client-deletion:
+    initiating-requests:
+      # [all oam requests]
```

Comment:
Along with the new add-ons, can we add a comment like [all oam put requests] to "initiating-requests" of OamRequestCausesLtpUpdateRequest forwarding